sdk-actors/src/main/java/io/dapr/actors/runtime/ActorRuntimeContext.java

```diff
@@ -21,7 +21,7 @@
   private final ActorRuntime actorRuntime;
 
   /**
-   * Serializer.
+   * Serializer for transient objects.

```

Comment:
I don't think this term has an obvious meaning and can be easily misinterpreted.

Reply:
Let me think about something else.

Reply:
See the latest version, please.